src/instructlab/cli/process/prune.py

```diff
@@ -0,0 +1,41 @@
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.process.process import (
+    filter_process_record_with_conditions,
+    remove_process_record,
+)
+
+
+@click.command(name="prune")
+@click.option(
+    "--older",
+    type=int,
+    required=True,
+    default=7,
+    help="Remove completed processes older than the specified number of days (defautl: 7 days).",
+)
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    help="Used to forcefully remove process record. Use with caution as it permanently deletes the record from the file without further confirmation.",
```

Comment:
"from the file" which file? user doesn't know and doesn't care since it's implementation detail. Let's not mention a file. I'd also not refer to these as "records" because from user perspective it's just an entry in a list of processes. (a process).

Reply:
ok